ext/xml/xml.c

```diff
@@ -820,8 +819,8 @@ void _xml_characterDataHandler(void *userData, const XML_Char *s, int len)
 }
 /* }}} */
 
-/* {{{ _xml_processingInstructionHandler() */
-void _xml_processingInstructionHandler(void *userData, const XML_Char *target, const XML_Char *data)
+/* {{{ xml_processingInstructionHandler() */
```

Comment:
nit: if I were you I would not bother with those maybe you can just remove them ?

Reply:
Maybe, I didn't really have a problem with them as this commit was just a bunch of ctrl+replace anyway
If they are to be removed I'd do it in one go for the entire extension